specification/billing/resource-manager/Microsoft.Billing/preview/2024-08-02-preview/migration.json

```diff
@@ -0,0 +1,204 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BillingManagementClient",
+    "description": "Documentation for Microsoft.Billing.",
+    "contact": {
+      "name": "Microsoft.Billing",
+      "url": "https://learn.microsoft.com/en-us/rest/api/billing/",
+      "email": "pacebpexphot@microsoft.com"
+    },
+    "version": "2024-08-02-preview"
+  },
+  "paths": {
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/migrations/default": {
+      "get": {
+        "tags": [
+          "Migration"
+        ],
+        "description": "Gets migration mapping details.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
+        },
+        "operationId": "Migration_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./types.json#/parameters/billingAccountName"
+          },
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          }
```

Comment:
Have you considered leveraging common types?

Reply:
Yes. "x-ms-parameter-location": "method" is missing in common type. Hence we have our version of apiVersionParameter